analyzer/operations.go

```diff
@@ -410,6 +410,7 @@ func ReconcileLogOpsWithTransfers(logOps, transferOps []Operation, tobinTax *Tob
 	for index := range logOps {
 		logOp := logOps[index]
 		if logOp.Type.requiresTransfer() {
+			//nolint:gosec
```

Comment:
I have no clue what's going on with this one tbh
we should investigate in future

Reply:
yeah, thanks for pointing out actually also a merge conflict that was silently resolved weirdly, fixing now (no longer need the nolint) -- a bit of a tricky loop though as `logOp` gets modified but in such a way that I think a shallow copy suffices. Def wanna understand this better though